src/Data/Nat/Properties.agda

```diff
@@ -1610,12 +1614,11 @@ m≤n⇒n∸m≤n (s≤s m≤n) = m≤n⇒m≤1+n (m≤n⇒n∸m≤n m≤n)
 ∸-+-assoc (suc m) (suc n) o = ∸-+-assoc m n o
 
 +-∸-assoc : ∀ m {n o} → o ≤ n → (m + n) ∸ o ≡ m + (n ∸ o)
-+-∸-assoc m (z≤n {n = n})             = begin-equality m + n ∎
-+-∸-assoc m (s≤s {m = o} {n = n} o≤n) = begin-equality
-  (m + suc n) ∸ suc o  ≡⟨ cong (_∸ suc o) (+-suc m n) ⟩
-  suc (m + n) ∸ suc o  ≡⟨⟩
-  (m + n) ∸ o          ≡⟨ +-∸-assoc m o≤n ⟩
-  m + (n ∸ o)          ∎
++-∸-assoc zero    {n = _} {o = _} _   = refl
```

Comment:
And in the spirit of the original proof:
```suggestion
+-∸-assoc zero    {n = n} {o = o} _   = begin-equality n ∸ o ∎
```
This is pure cosmetics, however, and I'm not really sure that `begin-equality x ∎` is quite the best way to write `refl` when you want to explicitly mention `x`?